lib/tether/tether.go

```diff
@@ -251,20 +252,32 @@ func (t *tether) setNetworks() error {
 }
 
 func (t *tether) setMounts() error {
+	// provides a lookup from path to volume reference.
+	NameLookupMap := make(map[string]string, 0)
```

Comment:
[cosmetic] NameLookupMap -> l maybe? NameLookupMap creates an illusion that map itself is exported